pkg/agent/agent_windows.go

```diff
@@ -104,9 +111,17 @@ func (i *Initializer) prepareHostNetwork() error {
 	return util.PrepareHNSNetwork(subnetCIDR, i.nodeConfig.NodeTransportIPv4Addr, adapter, i.nodeConfig.UplinkNetConfig.Gateway, dnsServers, i.nodeConfig.UplinkNetConfig.Routes, i.ovsBridge)
 }
 
-// prepareOVSBridge adds local port and uplink to ovs bridge.
-// This function will delete OVS bridge and HNS network created by antrea on failure.
+// prepareOVSBridge operates OVS bridge
```

Comment:
I feel we can remove this line, as "operates OVS bridge" is not more accurate than "prepareOVSBridge".

Reply:
done